### DISCUSSION TITLE
The PHP OPcache module is not properly configured - fix?

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#2795] [Questions] [answered]
I have the Security & setting warning on my Nextcloud AIO docker 26.0.2 (Ubuntu server 22.04.6)

`There are some warnings regarding your setup.
 The PHP OPcache module is not properly configured. The OPcache buffer is nearly full. To assure that all scripts can be hold in cache, it is recommended to apply opcache.memory_consumption to your PHP configuration with a value higher than 128.`

How can I fix it?

**Comment by szaimen** [comment, marked as answer]
Hi, this was already adjusted with https://github.com/nextcloud/all-in-one/pull/2762 and will be fixed in the next version.

**Reply by szaimen** [reply]
This is now released with v6.2.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by vawaver** [reply]
Thank you for info but I will stay on `stable` channel.

**Comment by simon-77** [comment]
I also have this error after upgrading to Nextcloud 30 (beta) today:


> The PHP OPcache module is not properly configured. The OPcache buffer is nearly full. To assure that all scripts can be hold in cache, it is recommended to apply "opcache.memory_consumption" to your PHP configuration with a value higher than "256".. For more details see the [documentation ↗](https://docs.nextcloud.com/server/30/go.php?to=admin-php-opcache).


Is it possible to change this cache value for my instance persistently, or can it only be changed with a new AIO version?

---

Nextcloud AIO v9.6.0
Nextcloud Hub 9 (30.0.1 RC1)

**Reply by szaimen** [reply]
@MichaIng i thought we got rid of this issue already a few versions ago? Is this a regression?

**Reply by simon-77** [reply]
The last warning (post from June) said the memory should be set larger than **128**
My warning says the memory should be set larger than **256**

... so maybe it's time to increase the memory again :shrug:

btw. I am using the Nextcloud Assistant with OpenAI integration (using the OpenAI API)

**Reply by MichaIng** [reply]
Is there an easy way to add e.g. https://github.com/amnuts/opcache-gui to the container and check OPcache usage, in case the particular scripts/apps in detail?

I'll also update one of my NC instances now to 30 beta and see whether it significantly raises.

EDIT: Here is my OPcache usage, after updating from NC30 beta (forgot I was on an older beta already) to RC1, and looping though all apps and settings pages:

![image](https://github.com/user-attachments/assets/6d9c8c2a-c190-488f-a187-caf8df074732)

Last time I checked, maybe NC28 or NC29, it more more like 50% usage, so raised a little, but not significantly.

**Comment by ic3-neo** [comment]
Hi,

I got 

`There are some warnings regarding your setup.

    The PHP OPcache module is not properly configured. The OPcache buffer is nearly full. To assure that all scripts can be hold in cache, it is recommended to apply "opcache.memory_consumption" to your PHP configuration with a value higher than "256".. For more details see the [documentation ↗](https://docs.nextcloud.com/server/31/go.php?to=admin-php-opcache).`

How I can get the usage as @MichaIng ?
